package-lock.json

```diff
@@ -0,0 +1,3999 @@
+{
```

Comment:
please dont commit this file as we are using yarn for builds and project already has yarn.lock file

Reply:
I need the body-parser dependency to read the req.body/payload to /api/bookings. I tried using the following to replace body-parser:

var data = "";
    req.on('data', function (chunk) { data += chunk })
    req.on('end', function () {
      req.rawBody = data;
      req.jsonBody = JSON.parse(data);
      const newBooking = req.jsonBody;
    }) 

But it does not recognize const newBooking elsewhere in the function, and putting the rest of the queries inside the req.on does not work. So I think I need to add body-parser in order for the route to work. 

Also, if we can't make changes to package.json, how will we be able to add socket.io (or long-polling)?

Reply:
You can make changes to package.json bt don't commit package-lock.json. this creates conflict while making build